### ISSUE TITLE
Tracking changes removing is not working correctly in review mode

### ISSUE THREAD

**Issue by aleksinpavel** [ONLYOFFICE/DocumentServer#658]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Removing added tracking changes in review mode is not working.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
1. Go to https://api.onlyoffice.com/editors/editor?method=docxReview
2. Add some text in review mode
3. Try to delete it
<img width="1440" alt="tracking changes removing" src="https://user-images.githubusercontent.com/12109564/65940718-df284700-e431-11e9-8b39-6a6c16d471b8.png">

**What is the expected behavior?**
Added tracking change removed completely.

**Did this work in previous versions of DocumentServer?**
On version 5.2.3-64 it’s working correctly.

**DocumentServer version:**
5.4.0-21

**Operating System:**
macOS Mojave 10.14.6

**Browser version:**
Google Chrome 77.0.3865.90 (64 bit)

**Comment by Rita-Bubnova** [comment]
Hello, @aleksinpavel.
This will be fixed in 5.4.1, that we hopefully release within a week.

**Comment by ShockwaveNN** [comment]
DocumentServer 5.4.1 is released. Issue should be fixed

**Comment by aleksinpavel** [comment]
This bug is still reproducible on 5.4.1.39. But right now it works incorrect only after saving document with some additions.

**Steps to reproduce:**
1. Open some document and make some text additions in review mode
2. Save the document and close it
3. Open this document after saving with the same user and try to delete added text
4. Same problem as described at this issue appears

**Comment by Rita-Bubnova** [comment]
Sorry, I cannot reproduce your problem.
Could you create a new issue and describe the steps for playback in more detail or сan you attach video of this problem?
